### PR TITLE
Add my videos entrypoint and related files

### PR DESCRIPTION
**Pull request by AdmiralCoco**

### CODE REVIEW

File: src/d2l-capture-central-app.js [hunk 1]
```diff
@@ -136,6 +137,9 @@ class D2lCaptureCentralApp extends DependencyRequester(NavigationMixin(InternalL
 						import('./pages/404/d2l-capture-central-404.js');
 						return;
 				}
+			case pageNames.myVideos:
+				import('./pages/my-videos/d2l-capture-central-my-videos.js');
```
Comment (qwangd2l) [inline src/d2l-capture-central-app.js:141]: I guess this file will be added in another PR or is it just missing in this PR?

**Review by qwangd2l — APPROVED** [review]

File: src/d2l-capture-central-app.js (hunk 1 above)
Comment (AdmiralCoco) [reply src/d2l-capture-central-app.js:141]: Ah yep it's added in the next PR... 🙂
